docs/management/admin/response-actions-history.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[response-actions-history]]
+= Response actions history
+
+{elastic-defend} keeps a log of the <<response-actions,response actions>> performed on endpoints, such as isolating a host or terminating a process. The log displays when each command was performed, the host on which the action was performed, the {kib} user who requested the action, any comments added to the action, and the action's current status.
+
+To access the response actions history for all endpoints, go to *Manage* -> *Response actions history*. You can also access the response actions history for an individual endpoint from these areas:
+
+* *Endpoints* page: Click an endpoint's name to open the details flyout, then click the *Response actions history* tab.
+* *Response console* page: Click the *Response actions history* button.
+
+All of these contexts contain the same information and features. The following image shows the *Response actions history* page for all endpoints:
+
+[role="screenshot"]
+image::images/response-actions-history-page.png[Response actions history page UI]
+
+Use these features to filter and expand the information in the response actions history:
+
+* Enter a user name or comma-separated list of user names in the search field to display actions requested by those users.
+* Use the *Hosts* menu to display actions performed on specific endpoints. (This menu is only available on the *Response actions history* page for all endpoints.)
+* Use the *Actions* menu to display specific actions types.
+* Use the *Statuses* menu to display actions with a specific status.
+* Use the date and time picker to display actions within a specific time range.
+* Click the expand arrow on the right to display more details about an action.
```

Comment:
```suggestion
* Click the expand button on the right to display more details about an action.
```

Reply:
I can't remember if this should be a button or icon.

Reply:
It's an icon because there's no text next to it, but I think "arrow" is a little more helpful because it's descriptive. I don't think [our guideline](https://docs.elastic.dev/content-design/tech-docs/writing-about-ui#icon-or-button) is saying that we _must only_ call it an icon, just that we shouldn't call it a button.